org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -0,0 +1,187 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+/* jshint sub:true */
+// This code uses ES2015 syntax that requires at least Node.js v4.
+// For Node.js ES2015 support details, reference http://node.green/
+
+"use strict";
+var request = require('request-promise');
+
+/**
+* This translator class implements the "Hub" interface.
+*/
+class Translator {
+    constructor(accessToken) {
+        this._accessToken = accessToken;
+
+        this._baseUrl = "https://api.wink.com";
+        this._devicesPath = '/users/me/wink_devices';
+
+        this._name = "Wink Hub"; // TODO: Can be pulled from OpenT2T global constants. This information is not available, at least, on wink hub.
+    }
+
+    /**
+     * Get the list of devices discovered through the hub.
+     */
+    getDevicesAsync() {
```

Comment:
Are you sure this Async prefix is supported? I just do get[PropertyName] so in this case I would have done getDevices().  See here for [example](https://github.com/openT2T/translators/blob/dev/org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/thingTranslator.js#L122). Maybe this is a bug in my translators? @jasongin what's the correct pattern here?